pkg/resourcecreator/testdata/naisjob/cronjob_timezone.yaml

```diff
@@ -0,0 +1,46 @@
+testconfig:
+  description: vanilla application gets a custom container command
```

Comment:
While the description is correct for the test as of writing, I suspect it isn't what was intended with this test 😛

Reply:
Whoops :blush:  Forgot `CTRL+S`